template-parts/content-home-list.php

```diff
@@ -27,7 +27,7 @@
 			<a href="<?php the_permalink(); ?>" rel="bookmark">
 				<?php the_post_thumbnail( 'medium' ); ?>   
 			</a>  
-			<?php echo loose_post_format_icon( get_the_ID() ); // WPCS: XSS OK. ?>
+			<?php echo esc_html( loose_post_format_icon( get_the_ID() ) ); ?>
```

Comment:
@DivyanshKhanna26 I bet that `post_format_icon` will output some html -> img tag.